src/modules/ekf2/ekf2_main.cpp

```diff
@@ -1175,7 +1189,7 @@ void Ekf2::run()
 				global_pos.vel_e = lpos.vy; // Ground east velocity, m/s
 				global_pos.vel_d = lpos.vz; // Ground downside velocity, m/s
 
-				global_pos.yaw = lpos.yaw; // Yaw in radians -PI..+PI.
+				global_pos.yaw = matrix::Eulerf(q).psi(); // Yaw in radians -PI..+PI.
```

Comment:
Why keep global yaw?

Reply:
It's used system wise, and didn't want to increment the PR more than it is now.